fre/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '2024.01'

```

Comment:
Is this used? I think it makes sense to store the version here instead of setup.py, but want to avoid duplication.

Reply:
i think the idea is to check the `fre-cli` version below with a pythonic approach, but i'm wondering if this idea is actually dead-on-arrival the more i think about it...